config/500-controller.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
   labels:
     contrib.eventing.knative.dev/release: devel
     control-plane: github-controller-manager
+    sources.knative.dev/service: github
```

Comment:
I assumed you were going to put this label on the instance resource of the GitHubSource. What is the value of putting it on the controller and various resources that are used to install it? Things like mink and our other ways of combining reconcilers make this fairly low value.